perfmetrics/scripts/ml_tests/tf/resnet/setup_scripts/setup_container.sh

```diff
@@ -49,6 +49,11 @@ then
   TEST_BUCKET="gcsfuse-ml-data-hns-central1"
   echo "enable-hns: true" >> /tmp/gcsfuse_config.yaml
   DIR=${DIR}_${BUCKET_TYPE}
+elif [ ${BUCKET_TYPE} == "non-hns" ];
```

Comment:
How many types of bucket do we support currently? If only two "hns" and "non-hns" then we don't need to add elif right?

Reply:
Currently,we support only 2. Should I remove the elif , I added it just in case bucket type which is passed to this script  is any invalid value/nil

Reply:
Okay, we can keep it. Just add a comment here, as it doesn't look logical to add gRPC config for non-hns bucket.

Reply:
Added.